### ISSUE TITLE
Cannot set locale to language "English (U.S.)"

### ISSUE THREAD

**Issue by DeeDeeRanged** [pyfa-org/Pyfa#2314]
## Bug Report
Cannot set locale to language "English (U.S.)"
locale 'en_US' cannot be set

### Expected behavior:
No error with startup.

### Actual behavior:
Pyfa error
Cannot set locale to language "English (U.S.)"
locale 'en_US' cannot be set

### Detailed steps to reproduce:
launch pyfa

### Fits involved in EFT format (Edit > To Clipboard > EFT):
n/a

### Release or development git branch? Please note the release version or commit hash:
Pyfa-2.3.0/2.31.0/2.32.0/2.32.1/2.33.0
Pyfa-2.29.0 works fine.

### Operating system and version (eg: Windows 10, OS X 10.9, OS X 10.11, Ubuntu 16.10):
Debian testing
[log.txt](https://github.com/pyfa-org/Pyfa/files/5797301/log.txt)

### Other relevant information:
Is there a way to remove localization like it was before Pyfa-2.3.0?

**Comment by zhaoweny** [comment]
You are launching pyfa from it's git root, is that right? 

I would suggest you try instructions from `locale/README.md`, where there are alternate ways to compile translations. And for `en_US` specifically, there are additional step to generate the locale `mo` file, please reference `.travis.yml`

**Comment by DeeDeeRanged** [comment]
I use the tarball on the releases page not the git download.

**Comment by DeeDeeRanged** [comment]
I expect Pyfa to set en_US automatically as default and only issues an error if it cannot set a chosen language.

**Comment by zhaoweny** [comment]
Yeah, you are running the "source code tarball" which is equivalent to running from git root.

**Comment by zhaoweny** [comment]
I think I have a solution for this. I think the problem is that your debian does not have en_US.UTF-8 locale enabled. You can fix this by these steps:

1. edit the file `/etc/locale.gen`, find the line of `# en_US.UTF-8 UTF-8` , remove `# ` part
2. re-run `locale-gen` to regenerate locales.

I'm testing this with a Debian buster VM running on hyper-v, locale set to en_HK. After these steps, `python3 pyfa.py` did not report errors.

I'm not so sure about `msgfmt.py` error

**Comment by zhaoweny** [comment]
And pls make sure you are running `python3 pyfa.py` with python 3.7. You can check this via `python3 --version`.

**Comment by blitzmann** [comment]
@DeeDeeRanged  thanks for the info. I've updated the README.

As for the local.gen file reverting, it looks like you might be able to run `sudo dpkg-reconfigure locales` after `locale-gen` and that might save the current setting? I'm going to close this, but if that works, please feel free to open a PR to include that in the README as well :)
